index.html

```diff
@@ -1250,7 +1250,7 @@ <h6 data-lang="zh-hant">發表對本文檔的建議</h6>
                   <p data-lang="zh-hant">文字以順時針方向旋轉90°，主要用於西文的單字詞、語句等。</p>
                 </li>
                 <li>
-                  <p data-lang="en">Set horizontally without changing orientation. This is usually applied to two-digit numbers or numbers with more than two digits.</p>
+                  <p data-lang="en">Set horizontally without changing orientation. This is usually applied to numbers comprising of two or more digits.</p>
```

Comment:
I would have said either:
'comprised of two or more', or
'comprising two or more'

Reply:
Agreed. Will make the change.